paymentsheet/src/main/java/com/stripe/android/link/injection/LinkModule.kt

```diff
@@ -28,4 +30,9 @@ internal interface LinkModule {
     @Binds
     @LinkScope
     fun bindsLinkAttestationCheck(linkAttestationCheck: DefaultLinkAttestationCheck): LinkAttestationCheck
+
+    companion object {
+        @Provides
+        fun provideLinkLaunchMode(): LinkLaunchMode? = null
+    }
```

Comment:
why is this needed?

Reply:
It'll be needed later. I accidentally included it while cherry-picking off [the dev branch](https://github.com/stripe/stripe-android/compare/lng/lc-consent-ui?expand=1).